econpizza/__init__.py

```diff
@@ -14,13 +14,15 @@
 from .solvers.solve_linear_state_space import solve_linear_state_space, find_path_linear_state_space
 from .solvers.shooting import find_path_shooting
 from .parser import parse, load
-
+from .config import config
 
 # set number of cores for XLA
 os.environ["XLA_FLAGS"] = f"--xla_force_host_platform_device_count={os.cpu_count()}"
 
 jax.config.update("jax_enable_x64", True)
 
+config.setup_persistent_cache()
```

Comment:
If `config.update` is the intented interface, then this should also be used to update folder paths (and automatically update `jax.config` if necessary).

Reply:
I think this can be achieved by calling the function in the setter of `enable_persistent_cache`. In this case even the function `setup_persistent_cache` should react only on `True` value of the option - meaning if the user sets it to `False` the folders should not get deleted but only future caching behavior should be turned off.